### PR TITLE
Mekhanik evgenii/fix 2256

### PR DESCRIPTION
**Pull request by EvgeniiMekhanik**

### CODE REVIEW

File: fw/http_tbl.c [hunk 1]
```diff
@@ -439,10 +439,12 @@ tfw_cfgop_http_rule(TfwCfgSpec *cs, TfwCfgEntry *e)
...
 	val = tfw_http_val_adjust(in_field_val, field,
 				  &val_len, &type_val, &op_val);
-	if (IS_ERR(val))
+	if (IS_ERR(val)) {
+		kfree(arg);
 		return PTR_ERR(val);
+	}
 
-	rule = tfw_http_rule_new(tfw_chain_entry, type, arg_size);
+	rule = tfw_http_rule_new(tfw_chain_entry, type, arg_size);;
```
Comment (const-t) [inline fw/http_tbl.c:447]: extra `;`

File: fw/http_tbl.c [hunk 2]
```diff
@@ -639,14 +640,14 @@ tfw_cfgop_release_rule(TfwHttpMatchRule *rule)
 {
 	if (rule->act.type == TFW_HTTP_MATCH_ACT_VHOST)
 		tfw_vhost_put(rule->act.vhost);
-	if (rule->val.type == TFW_HTTP_MATCH_V_COOKIE)
+	if (rule->val.type == TFW_HTTP_MATCH_V_COOKIE) {
```
Comment (const-t) [inline fw/http_tbl.c:643]: extra curly braces

**Review by const-t — APPROVED** [review]
LGTM

File: fw/http_tbl.c (hunk 1 above)
Comment (krizhanovsky) [reply fw/http_tbl.c:447]: Doesn't checkpatch.pl reveal the typo? @EvgeniiMekhanik could you please check your git setup

**Review by krizhanovsky — APPROVED** [review]
LGTM

**Pull request merged**